granulate_utils/linux/cgroups/__init__.py

```diff
@@ -10,6 +10,8 @@
 from .cpu_cgroup import CpuCgroup
 from .memory_cgroup import MemoryCgroup
 
+__all__ = ["CpuCgroup", "MemoryCgroup"]
```

Comment:
why?

Reply:
removed

Reply:
https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used